backend/supervisor.sh

```diff
@@ -26,7 +26,8 @@ start_server
 
 # Keep the script running and wait for file changes
 while true; do
-    inotifywait -e modify -r .
-    stop_server
-    start_server
+    #inotifywait -e modify -r .
```

Comment:
As mentioned above here https://github.com/openpredictionmarkets/socialpredict/pull/219/files#r1689635236 ... this is where the container looks for changes in order to re-write the Golang binary, to achieve dev mode container.